.github/workflows/latency-tracking.yml

```
@@ -1,16 +1,13 @@
 name: CI
 
 on:
-  push:
-    branches-ignore:
-      - '**'  # temporally ignore all
-
-#on:
-#  pull_request_target:
+  pull_request_target:
 
 jobs:
   latency-tracking:
     runs-on: ubuntu-latest
+    env:
+      NUM_LAST_RELEASE: 3
```

```suggestion
      NUM_LAST_RELEASE: 1
```
otherwise it takes too long